src/Categories/Category/Monoidal/Star-Autonomous.agda

```diff
@@ -0,0 +1,35 @@
+{-# OPTIONS --without-K --safe #-}
+
+open import Categories.Category
+open import Categories.Category.Monoidal.Core using (Monoidal)
+open import Categories.Category.Monoidal.Symmetric
+
+module Categories.Category.Monoidal.Star-Autonomous {o ℓ e} {C : Category o ℓ e} (M : Monoidal C) where
+
+open import Level
+
+open import Categories.Category.Product
+open import Categories.Functor
+open import Categories.Functor.Properties
+open import Categories.NaturalTransformation.NaturalIsomorphism
+open import Categories.Morphism C
+open import Categories.Functor.Hom
+open Hom C
+
+open Category C renaming (op to Cᵒᵖ) hiding (id)
+open Monoidal M
+open Functor ⊗ renaming (op to ⊗ₒₚ)
+
+open import Relation.Binary.Structures
+
+record Star-Autonomous : Set (levelOfTerm M) where
+  field
+    symmetric : Symmetric M
+    Star : Functor Cᵒᵖ C
+
+  open Functor Star renaming (op to Starₒₚ ; F₀ to Star₀)
```

Comment:
Some stuff like this should be opened `public`, or somehow available outside without manually opening `Functor` modules and stuff. This might be the point where you rename `F₀` to `_*₀` and `F₁` to `_*₁`, but all the properties should be exported somehow too.

Reply:
I see! I realized the second renaming is not used so I've deleted it. Also I'm not sure how to manually open  `Star` functor outside since it is defined.